Block/Adminhtml/Form/Field/Accounts.php

```diff
@@ -0,0 +1,199 @@
+<?php
+namespace Kevin\Payment\Block\Adminhtml\Form\Field;
+
+use Kevin\Payment\Block\Adminhtml\Form\Field\Accounts\CountryColumn;
+use Kevin\Payment\Block\Adminhtml\Form\Field\Accounts\BankColumn;
+use Magento\Config\Block\System\Config\Form\Field\FieldArray\AbstractFieldArray;
+use Magento\Framework\DataObject;
+use Magento\Framework\Exception\LocalizedException;
+
+class Accounts extends AbstractFieldArray
+{
+    /**
+     * @var \Kevin\Payment\Model\PaymentMethodsFactory
+     */
+    protected $paymentMethodsFactory;
+
+    /**
+     * @var \Magento\Framework\Serialize\Serializer\Json
+     */
+    protected $json;
+
+    /**
+     * @param \Magento\Backend\Block\Template\Context $context
+     * @param \Kevin\Payment\Model\PaymentMethodsFactory $paymentMethodsFactory
+     * @param \Magento\Framework\Serialize\Serializer\Json $json
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Backend\Block\Template\Context $context,
+        \Kevin\Payment\Model\PaymentMethodsFactory $paymentMethodsFactory,
+        \Magento\Framework\Serialize\Serializer\Json $json,
+        array $data = []
+    ){
+        $this->paymentMethodsFactory = $paymentMethodsFactory;
+        $this->json = $json;
+
+        parent::__construct($context, $data);
+    }
+
+    /**
+     * @var CountryColumn
+     */
+    private $countryRenderer;
+
+    /**
+     * @var BankColumn
+     */
+    private $bankRenderer;
+
+    /**
+     * Prepare rendering the new field by adding all the needed columns
+     */
+    protected function _prepareToRender()
```

Comment:
If we choose to rename the class, this method could be renamed to `buildView`.

Reply:
_prepareToRender is extended method from parent class, and we can't raname it.